### PR TITLE
dts: vendor-prefixes: Add Bang & Olufsen A/S

### PR DESCRIPTION
**Pull request by hansbinderup**
Add Bang & Olufsen vendor prefix.

### CODE REVIEW

**Comment by nordicjm** [comment]
Are you submitting a driver to zephyr?

File: dts/bindings/vendor-prefixes.txt [hunk 1]
```diff
@@ -95,6 +95,7 @@ bitmain	Bitmain Technologies
 blues	Blues Wireless
 blutek	BluTek Power
 boe	BOE Technology Group Co., Ltd.
+beo	Bang & Olufsen A/S
```
Comment (pdgendt) [inline dts/bindings/vendor-prefixes.txt:98]: Please keep the order alphabetically correct.

**Review by pdgendt — DISMISSED** [review]

**Comment by hansbinderup** [comment]
> Are you submitting a driver to zephyr?

Currently speaking, no, but we have the intention to upstream our drivers.

File: dts/bindings/vendor-prefixes.txt (hunk 1 above)
Comment (pdgendt) [reply dts/bindings/vendor-prefixes.txt:97]: Still not correct, it should be set after `beagle`.

**Comment by hansbinderup** [comment]
> > > Are you submitting a driver to zephyr?
> > 
> > 
> > Currently speaking, no, but we have the intention to upstream our drivers.
> 
> Looking forward to it! Please don't update the vendor prefixes file until that is the case (see [docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#id7](https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#id7))

Was missing that, sorry. Thanks for the reference! :+1: 
I'll close this PR for now then.

**Review by kartben — CHANGES_REQUESTED** [review]
Vendor prefixes file should not be updated without at least a new binding or driver using it
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-vendor-prefixes

**Comment by kartben** [comment]
> > > > Are you submitting a driver to zephyr?
> > > 
> > > 
> > > Currently speaking, no, but we have the intention to upstream our drivers.
> > 
> > Looking forward to it! Please don't update the vendor prefixes file until that is the case (see [docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#id7](https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#id7))
> 
> Was missing that, sorry. Thanks for the reference! :+1: 
> I'll close this PR for now then.

No worries!

**Pull request closed without merging**